dist/config/packages/monsieurbiz_sylius_rich_editor_plugin.yaml

```diff
@@ -0,0 +1,2 @@
+imports:
```

Comment:
It is but at the end, we will not have recipes files anymore when we will have the recipes on Symfony side.

Reply:
And I am not fond of a symlink which will be broken while it's not copied in the test application.
Because path in `test/Application` is `../../../../recipes/2.0-dev/config/packages/monsieurbiz_sylius_rich_editor_plugin.yaml` and in dist folder `../../../recipes/2.0-dev/config/packages/monsieurbiz_sylius_rich_editor_plugin.yaml`